book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_reader :rentals
```

Comment:
- [ ] Great job using getters for your rentals. however, please kindly take note that the project requirement is to use Getters and Setters for @rentals. please kindly make use of attr_accessor for your rentals also.